src/main/java/net/minecraftforge/registries/ForgeRegistry.java

```diff
@@ -112,7 +104,7 @@ public class ForgeRegistry<V> implements IForgeRegistryInternal<V>, IForgeRegist
         this.missing = builder.getMissingFactory();
         this.allowOverrides = builder.getAllowOverrides();
         this.isModifiable = builder.getAllowModifications();
-        this.hasWrapper = builder.getHasWrapper();
+        this.hasWrapper = true;
```

Comment:
Any reason why the field is still there?

Reply:
No, I thought I nuked it